### PR TITLE
Add og:image support and ability to extend behavior using events

### PR DESCRIPTION
**Pull request by hugochinchilla**
I did this modifications for a project and I think this may be useful for other people.

### CODE REVIEW

File: components/Seo.php [hunk 1]
```diff
@@ -53,7 +56,7 @@ public function onRender()
 
         if ($isBlogPage !== false) {
             $this->getDataFromBlogPost();
-        } else if ($isStaticPage !== false) {
+        } elseif ($isStaticPage !== false) {
```
Comment (Magiczne) [inline components/Seo.php]: I prefer to have `else if` than `elseif`

File: components/Seo.php [hunk 2]
```diff
@@ -72,6 +75,17 @@ private function getDataFromBlogPost()
             $this->redirectUrl = $post->seo_redirect_url;
             $this->robotsIndex = $post->seo_robots_index;
             $this->robotsFollow = $post->seo_robots_follow;
+
+            $featuredImage = $post->featured_images->first();
+            if ($featuredImage) {
+                $this->ogImage = $featuredImage->path;
+                $localPath = $featuredImage->getLocalPath();
+                if (is_file($localPath)) {
+                    list($width, $height) = getimagesize($localPath);
```
Comment (Magiczne) [inline components/Seo.php]: Please use short syntax

```php
[$width, $height] = getimagesize($localPath);
```

**Review by Magiczne — CHANGES_REQUESTED** [review]

**Comment by Magiczne** [comment]
Please add some information to the Readme about this functionality if you can. :)

File: components/Seo.php (hunk 1 above)
Comment (hugochinchilla) [reply components/Seo.php]: done

File: components/Seo.php (hunk 2 above)
Comment (hugochinchilla) [reply components/Seo.php]: done

**Comment by hugochinchilla** [comment]
I added some info in the README

**Review by Magiczne — APPROVED** [review]

**Comment by Magiczne** [comment]
Thanks! New version published.

**Pull request merged**